PACKAGES

```diff
@@ -45,3 +45,6 @@ ggrepel
 ggthemes
 knitr
 skimr
+shiny.i18n
```

Comment:
I think this line is a duplicate

Reply:
Sorry, didn't notice it was already there.
Should I resubmit a PR?

Reply:
No need! We'll squash this PR into one commit so should be no problem

Reply:
Didn't know it automatically detects additional commits.
I removed the entry.

Thanks!